interface/skin.dmf

```diff
@@ -1,494 +1,494 @@
+		category = "&Size"
+		can-check = true
+		group = "size"
+		saved-params = "is-checked"
+	elem "icon80"
+		name = "&80x80 (2.5x)"
+		command = ".winset \"paramapwindow.map.icon-size=80\""
+		category = "&Size"
+		can-check = true
+		group = "size"
+		saved-params = "is-checked"
+	elem "icon64"
+		name = "&64x64 (2x)"
+		command = ".winset \"paramapwindow.map.icon-size=64\""
+		category = "&Size"
+		can-check = true
+		group = "size"
+		saved-params = "is-checked"
+	elem "icon48"
+		name = "&48x48 (1.5x)"
+		command = ".winset \"paramapwindow.map.icon-size=48\""
+		category = "&Size"
+		can-check = true
+		group = "size"
+		saved-params = "is-checked"
+	elem "icon32"
+		name = "&32x32"
+		command = ".winset \"paramapwindow.map.icon-size=32\""
+		category = "&Size"
+		can-check = true
+		group = "size"
+		saved-params = "is-checked"
+	elem
+		name = "&Scaling"
+		command = ""
+		category = "&Icons"
+		saved-params = "is-checked"
+	elem "NN"
+		name = "&Nearest Neighbor"
+		command = ".winset \"paramapwindow.map.zoom-mode=distort\""
+		category = "&Scaling"
+		is-checked = true
+		can-check = true
+		group = "scale"
+		saved-params = "is-checked"
+	elem "PS"
+		name = "&Point Sampling"
+		command = ".winset \"paramapwindow.map.zoom-mode=normal\""
+		category = "&Scaling"
+		can-check = true
+		group = "scale"
+		saved-params = "is-checked"
+	elem "BL"
+		name = "&Bilinear"
+		command = ".winset \"paramapwindow.map.zoom-mode=blur\""
+		category = "&Scaling"
+		can-check = true
+		group = "scale"
+		saved-params = "is-checked"
+	elem "textmode"
+		name = "&Text"
+		command = ".winset \"menu.textmode.is-checked=true?paramapwindow.map.text-mode=true:paramapwindow.map.text-mode=false\""
+		category = "&Icons"
+		can-check = true
+		saved-params = "is-checked"
+	elem
+		name = "&Options"
+		command = ""
+		saved-params = "is-checked"
+	elem
+		name = "&Open Volume Mixer"
+		command = "Open-Volume-Mixer"
+		category = "&Options"
+		saved-params = "is-checked"
+	elem "statusbar"
+		name = "&Show status bar"
+		command = ".winset \"menu.statusbar.is-checked=true?paramapwindow.status_bar.is-visible=true:paramapwindow.status_bar.is-visible=false\""
+		category = "&Options"
+		is-checked = true
+		can-check = true
+		saved-params = "is-checked"
+	elem
+		name = "&Help"
+		command = ""
+		saved-params = "is-checked"
+	elem
+		name = "&Admin help"
+		command = "adminhelp"
+		category = "&Help"
+		saved-params = "is-checked"
+
+
+window "mainwindow"
+	elem "mainwindow"
+		type = MAIN
+		pos = 0,0
+		size = 640x440
+		anchor1 = -1,-1
+		anchor2 = -1,-1
+		is-default = true
+		saved-params = "pos;size;is-minimized;is-maximized"
+		title = "Paradise Station 13"
+		statusbar = false
+		is-maximized = true
+		icon = 'icons\\paradise.png'
+		macro = "default"
+		menu = "menu"
+	elem "mainvsplit"
+		type = CHILD
+		pos = 0,0
+		size = 0x0
+		anchor1 = 0,0
+		anchor2 = 100,100
+		saved-params = "splitter"
+		left = "rpane"
```

Comment:
It decided to replace the entire file for some reason, but the only real change in this file is here
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/2279af1b-5eaf-4112-8c60-5e338c98928e)